module/x/gravity/types/outgoing_tx.go

```diff
@@ -37,15 +37,15 @@ type ABIEncodedValsetArgs struct {
 
 // TODO: do we need a prefix byte for the different types?
 func (sstx *SignerSetTx) GetStoreIndex() []byte {
-	return MakeSignerSetTxKey(sstx.Nonce)
+	return MakeSignerSetTxKey(ChainIDOrDefault(sstx.GetChainId()), sstx.Nonce)
```

Comment:
Whoops, I was wrong, this is still being used, but we should delete the "default" thing and be explicit.